Source/Core/DolphinWX/MainAndroid.cpp

```diff
@@ -287,12 +437,12 @@ JNIEXPORT jboolean JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_Supports
 
 JNIEXPORT void JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_SaveScreenShot(JNIEnv *env, jobject obj)
 {
-	Core::SaveScreenShot();
+Core::SaveScreenShot();
```

Comment:
*sigh* I'm using Jetbrains' C++ IDE, which despite having a "1.0" version tag, mysteriously seems to have no concept whatsoever of what is going on with all these JNI methods. So the code style autoformatter seems to have barfed on it. I've already fixed this in the next PR I'm putting up.